workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/test-data/contrasts_files.txt

```diff
@@ -0,0 +1,2 @@
+Header
```

Comment:
How does a user know what to put in here ? Can we avoid this file ? Alternatively can this be a text input parameter ?

Reply:
We are working on it ;)! Thank you!

Reply:
Hi @mvdbeek ! We added an AWK code step. No file needed anymore =D